doc/changes/latest.inc

```diff
@@ -90,6 +92,8 @@ Enhancements
 
 - :func:`mne.time_frequency.EpochsTFR.average` now allows different ways of averaging, such as "median", or callable functions (:gh:`8879` by `Adam Li`_)
 
+- New Tutorial for analyzing frequency-tagging data (:gh: `8867` **by new contributor** |Dominik Welke|_ and `kalenkovich`_)
```

Comment:
Can you move this to the top of the `Enhancements` section? We list the changes by new contributors at the top of each section
```suggestion
- New Tutorial for analyzing frequency-tagging data (:gh:`8867` **by new contributor** |Dominik Welke|_ and `kalenkovich`_)
```
Also the space after the `:gh:` is causing problems for CircleCI